modules/core/src/v2/coins/algo.ts

```diff
@@ -1,35 +1,120 @@
 import BaseCoin = require('../baseCoin');
+import * as _ from 'lodash';
+
+const { MultiSigTransaction } = require('algosdk').Multisig;
```

Comment:
kind of funky to require algosdk twice (I know it's cached)

Reply:
ha, ok changed